Documentation/gettingstarted/ipvlan.rst

```diff
@@ -0,0 +1,116 @@
+datapath instead of the default veth-based one.
+
+.. note::
+
+    This is a beta feature. Please provide feedback and file a GitHub issue
+    if you experience any problems.
+
+First step is to download the Cilium Kubernetes descriptor:
+
+.. tabs::
+  .. group-tab:: K8s 1.13
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.13/cilium.yaml
+
+  .. group-tab:: K8s 1.12
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.12/cilium.yaml
+
+  .. group-tab:: K8s 1.11
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
+
+  .. group-tab:: K8s 1.10
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
+
+  .. group-tab:: K8s 1.9
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
+
+  .. group-tab:: K8s 1.8
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
+
+Edit the ``cilium-config`` ConfigMap in that file with the etcd server
+that is running in your cluster and set the option ``datapath-mode`` to
+``"ipvlan"``.
+
+It is also required to specify the ipvlan master device which typically
+points to a networking device that is facing the external network. This
+is done through setting ``ipvlan-master-device`` to the name of the
+networking device such as ``"eth0"`` or ``"bond0"``, for example. For
+deployment, this device must have consistent naming on each host.
+
+The ipvlan datapath only supports direct routing mode, therefore
+tunneling is disabled through setting ``tunnel`` to ``"disabled"``.
+
+The ``--install-iptables-rules`` parameter is optional and if set to
+``"false"`` then Cilium will not install any iptables rules which are
+mainly for interaction with kube-proxy, and additionally it will trigger
+ipvlan setup in L3 mode. For the default case where the latter is ``"true"``,
+ipvlan is operated in L3S mode such that netfilter in host namespace
+is not bypassed. Optionally, the agent can also be set up for masquerading
+all traffic leaving the ipvlan master device if ``masquerade`` is set
+to ``"true"``.
+
+Example ConfigMap extract for ipvlan in pure L3 mode:
+
+::
+
+  datapath-mode: "ipvlan"
+  ipvlan-master-device: "bond0"
+  tunnel: "disabled"
+  install-iptables-rules: "false"
+
+Example ConfigMap extract for ipvlan in L3S mode with masquerading
+all traffic leaving the node:
+
+::
+
+  datapath-mode: "ipvlan"
+  ipvlan-master-device: "bond0"
+  tunnel: "disabled"
+  masquerade: "true"
+
+Apply the DaemonSet file to deploy Cilium and verify that it has
+come up correctly:
+
+.. parsed-literal::
+
+    kubectl create -f ./cilium.yaml
+    kubectl -n kube-system get pods -l k8s-app=cilium
+    NAME                READY     STATUS    RESTARTS   AGE
+    cilium-crf7f        1/1       Running   0          10m
+
+In order for L3S mode to work correctly, a kernel with the following fix
```

Comment:
I think this warning / node should be when you are explaining that L3S is available. It's frustrating for a user to follow a tutorial, in this case with L3S enabled and then realize that he has to start over because he does not have the necessary requirements.